theme/src/components/search.js

```diff
@@ -14,6 +14,7 @@ function Search(props) {
         {...getInputProps({
           placeholder: `Search ${siteMetadata.title}`,
           width: 240,
+          'aria-label': `Search ${siteMetadata.title}`
```

Comment:
There is another search input in `theme/src/components/mobile-search.js`. Can you add the `aria-label` there as well?